BrowserExtension/scripts/community/inventory.js

```diff
@@ -18,6 +18,218 @@
+	var originalPopulateMarketActions = window.PopulateMarketActions;
+	
+	var dummySellEvent =
+	{
+		stop: function()
+		{
+			
+		}
+	}
+	
+	var quickSellButton = function( )
+	{
+		SellCurrentSelection();
+		
+		document.getElementById( 'market_sell_currency_input' ).value = this.dataset.price;
+		document.getElementById( 'market_sell_dialog_accept_ssa' ).checked = true;
+		
+		SellItemDialog.OnInputKeyUp( null ); // Recalculate prices
+		SellItemDialog.OnAccept( dummySellEvent );
+		
+		if( document.body.dataset.steamdbQuickSellAuto )
+		{
+			SellItemDialog.OnConfirmationAccept( dummySellEvent );
+		}
+	};
+	
+	if( document.body.dataset.steamdbNoSellReload )
+	{
+		var nextRefreshCausedBySell = false;
+		var originalOnConfirmationAccept = window.SellItemDialog.OnConfirmationAccept;
+		var originalReloadInventory = window.CUserYou.prototype.ReloadInventory;
+		
+		window.SellItemDialog.OnConfirmationAccept = function( event )
+		{
+			nextRefreshCausedBySell = true;
+			
+			originalOnConfirmationAccept.apply( this, arguments );
+		};
+		
+		window.CUserYou.prototype.ReloadInventory = function( appid, contextid )
+		{
+			if( nextRefreshCausedBySell )
+			{
+				nextRefreshCausedBySell = false;
+				
+				window.g_ActiveInventory.selectedItem.element.style.opacity = 0.2;
+			}
+			else
+			{
+				originalReloadInventory.apply( this, arguments );
+			}
+		};
+	}
+	
+	window.PopulateMarketActions = function( elActions, item )
+	{
+		var realIsTrading       = window.g_bIsTrading;
+		var realIsMarketAllowed = window.g_bMarketAllowed;
+		
+		if( !window.g_bViewingOwnProfile )
+		{
+			window.g_bIsTrading     = true; // Hides sell button
+			window.g_bMarketAllowed = true; // Has to be set so Valve's code doesn't try to bind a tooltip on non existing sell button
+		}
+		
+		originalPopulateMarketActions.apply( this, arguments );
+		
+		window.g_bIsTrading     = realIsTrading;
+		window.g_bMarketAllowed = realIsMarketAllowed;
+		
+		try
+		{
+			if( hasQuickSellEnabled && item.marketable && !item.is_currency && elActions.style.display !== 'none' )
+			{
+				var buttons = document.createElement( 'span' );
+				buttons.style.float = 'right';
+				
+				var sellNowText;
+				var listNowText = document.createElement( 'span' );
+				listNowText.textContent = 'List now (…)';
+				
+				var sellNow = null;
+				var listNow = document.createElement( 'a' );
+				listNow.href = 'javascript:void(0)';
+				listNow.className = 'btn_small btn_blue_white_innerfade';
+				listNow.style.opacity = 0.5;
+				listNow.appendChild( listNowText );
+				
+				buttons.appendChild( listNow );
+				
+				if( item.commodity )
+				{
+					sellNowText = document.createElement( 'span' );
+					sellNowText.textContent = 'Sell now (…)';
+					
+					sellNow = document.createElement( 'a' );
+					sellNow.href = 'javascript:void(0)';
+					sellNow.className = 'btn_small btn_blue_white_innerfade';
+					sellNow.style.opacity = 0.5;
+					sellNow.appendChild( sellNowText );
```

Comment:
Perhaps it would be good to add a title to these buttons to clarify the difference between "List Now" and "Sell Now".